### ISSUE TITLE
Colors are not supported

### ISSUE THREAD

**Issue by dmitry-saritasa** [mantoni/eslint_d.js#88]
### eslint_d output

```eslint_d . --cache --ext .js --ext .vue```

![image](https://user-images.githubusercontent.com/1747653/41882219-9823a860-789c-11e8-9c32-022acc0d00b9.png)

### eslint output

```eslint . --cache --ext .js --ext .vue```

![image](https://user-images.githubusercontent.com/1747653/41882235-ac66c834-789c-11e8-97fa-0eeabdc9686c.png)

### versions

```
➜  frontend git:(develop) ✗ npx eslint_d --version                                 
v4.19.1 (eslint_d v5.3.1)

~/code/fiq/frontend develop*
➜  frontend git:(develop) ✗ npx eslint --version  
v4.19.1

~/code/fiq/frontend develop*
➜  frontend git:(develop) ✗ node --version                                         
v8.11.2

~/code/fiq/frontend develop*
➜  frontend git:(develop) ✗ npm --version 
5.5.1
```

**Comment by mantoni** [comment]
Thank you for the detailed report. There are actually two independent problems with color support:

1. Color support is detected to be `false` by `chalk` because the stdout stream of the background process is not a tty.
2. There is [this bug in chalk](https://github.com/chalk/chalk/issues/234) which means that `--color=true` does not work either.

To solve the first issue, we'd need to detec color support in the client and tell the server through the internal protocol whether colors should be enabled or not. However, this would most probably lead to the second issue because we cannot enable color support once detected as false.

To fix this, we need the chalk issue to be resolved, which allows you to use `--color=true` as a workaround. Once that is fixed, we can think about passing color support from the client to the server.

Help on this is greatly appreciated. I will not have the time to look into this myself right now.

**Comment by nazrhyn** [comment]
@mantoni I noticed your reference to my issue and came over to read this one. I wanted to make a slight correction to what you wrote, here:

The `--color|--colors|--color=true|--color=always` option/argument/flag is actually processed immediately by the **supports-color** library (https://github.com/chalk/supports-color/blob/master/index.js#L12), so adding this to the command that runs your program actually _does_ work to force chalk colors on. (e.g. `node program.js -- --color`)

The issue I submitted is about being able to force it on during run-time, _after_ TTY detection has determined that colors are not supported, using either the `enabled` flag or `{ enabled: true }` in the options (https://github.com/chalk/chalk#chalkenabled).

That may have been what you meant, but I wanted to make sure there wasn't any confusion! :wave:

**Comment by mantoni** [comment]
@nazrhyn Thank you for the clarification. My statement that "`--color=true` does not work" isn't correct, indeed. However, we also have the issue that we'd want to enable / disable color in the running process.

Your comment lead me to dig a little bit deeper into this and I found an interesting workaround: I can force color when starting `eslint_d` like this:

```bash
eslint_d stop
export FORCE_COLOR=1
eslint_d start
```

This leads to colored output for me AND I can disable color by calling `eslint_d --no-color .`. So a possible solution could be to always force color on the background process.

**Comment by nazrhyn** [comment]
Great! I'm glad my comment was useful :blush:.

**Comment by mantoni** [comment]
Fix released in `v6.0.1`. Thanks everyone.
